pages/common/ibmcloud.md

```diff
@@ -0,0 +1,24 @@
+# ibmcloud
+
+> A command line tool for managing IBM Cloud apps and services.
+> More information: <https://ibm.biz/cli-docs>.
+
+- Show help for a specific subcommand (use without subcommand for general help):
+
+`ibmcloud help {{subcommand}}`
+
+- Show ibmcloud version:
```

Comment:
In my opinion this is an unnessecary example.

Reply:
OK, I can see why. I based this on the git page, as recommended in the CONTRIBUTING document.

Reply:
Depending on the command, it can be useful to know which version you are using. We have a `version` exaple in many pages. However we always keep it at the botton